src/legacy/core_plugins/timelion/public/timelion_vis_fn.ts

```diff
@@ -77,14 +78,14 @@ export const getTimelionVisualizationConfig = (
       forceFetch: true,
     });
 
-    response.visType = 'timelion';
+    response.visType = TIMELION_VIS_NAME;
 
     return {
       type: 'render',
       as: 'visualization',
       value: {
         visParams,
-        visType: name,
```

Comment:
Originally we had `timelion_vis` here.  It confuse a little. I think we should use one name for vis type and for build pipeline loader. What about to rename it in` build_pipeline.ts (line 252)` too?

Reply:
not possible, `timelion` expression function name is already taken, renaming `timelion` visualization to `timelion_vis` would break existing saved objects.